app/controllers/podcast_episode_appearances_controller.rb

```diff
@@ -0,0 +1,114 @@
+class PodcastEpisodeAppearancesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create edit update destroy]
+  before_action :set_episode, only: %i[new]
+  before_action :set_podcast_episode_appearance, only: %i[show edit update destroy]
+  before_action :set_episode_with_strong_params, only: %i[create update]
+  before_action :set_podcast_episode_appearances, only: %i[index create update destroy]
+
+  def index; end
+
+  def show; end
```

Comment:
Similar as to the `index`, what I usually think about when designing features like this is: where do I show this? Where do we need to show a single row of `PodcastAppearance` (which is a `m2m` table) ? 

In this case nowhere. The information itself it's already part of listing the users that appeared in  an episode in the episode page